internal/handlers/common/msg_debugerror.go

```diff
@@ -18,11 +18,27 @@ import (
 	"context"
 	"errors"
 
+	"github.com/FerretDB/FerretDB/internal/util/lazyerrors"
 	"github.com/FerretDB/FerretDB/internal/wire"
 )
 
 // MsgDebugError is a common implementation of the debugError command.
 func MsgDebugError(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg, error) {
 	// TODO https://github.com/FerretDB/FerretDB/issues/640
-	return nil, errors.New("debugError")
+	document, err := msg.Document()
+	if err != nil {
+		return nil, lazyerrors.Error(err)
+	}
+
+	expected, err := GetRequiredParam[string](document, document.Command())
+	if err != nil {
+		return nil, err
+	}
+
+	switch expected {
+	case "panic":
+		panic("oops!")
+	default:
+		return nil, errors.New(expected)
+	}
```

Comment:
We'd want a test each cases.

Reply:
Sure, that's for someone who would actually do that task :) That's just a start to make it easier to estimate

Reply:
Ohh, it almost looked like this task was done :see_no_evil: